dojo/db_migrations/0009_auto_20190621_1445.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.1 on 2019-06-21 14:45
```

Comment:
There is no need for two migrations that do the same things. Will most likely result in errors.